addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -1356,7 +1365,19 @@ export class Rtc extends Record {
             audioTrack.enabled = !this.selfSession.isMute && this.selfSession.isTalking;
             this.state.audioTrack = audioTrack;
             this.linkVoiceActivationDebounce();
-            await this.network?.updateUpload("audio", this.state.audioTrack);
+            if (this.state.screenAudioTrack) {
+                const audioContext = new AudioContext();
```

Comment:
At some point the `AudioContext` should be [closed](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext/close), when a new one is made, and in `clear()`